Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___PresenterSpec.swift

```diff
@@ -15,11 +15,11 @@ final class ___VARIABLE_moduleName___PresenterSpec: QuickSpec {
 
     override func spec() { 
 
-        var presenter: ___VARIABLE_moduleName___Presenter!
-        var router: ___VARIABLE_moduleName___RouterInputMock!
-        var interactor: ___VARIABLE_moduleName___InteractorInputMock!
-        var view: ___VARIABLE_moduleName___ViewInputMock!
-        var output: ___VARIABLE_moduleName___OutputMock!
+        private var presenter: ___VARIABLE_moduleName___Presenter!
```

Comment:
No need to do this.

Reply:
@nmint8m Please give me a reason that `private` is same with `internal` in this case

Reply:
You define this inside function `spec()`, don't you?.